### PR TITLE
#170947627: As a travelAdmin or a Pre screened suppliers I should be able to create Accommodation

### PR DESCRIPTION
**Pull request by Kagorora**
#### What does this PR do?

- Will allow Travel Admin or Pre Pre-screened suppliers to create or add accommodation to the system.

#### Description of Task to be completed?

- [x] Design the mockup

- [x] Design the UI

- [x] Validate Inputs

- [x] Add Functionality to create accommodation

- [x] Test endpoint

#### How should this be manually tested?

- Clone the project

       git clone https://github.com/Stackup-Rwanda/knights-bn-frontend.git

- Move to the branch

       git checkout ft-create-Accommodation-170947627

-  Install dependencies

       npm install

-  start the server

       npm run dev-start 

-  log in as a travel admin

        email: traveladmin@gmail.com
        password: Niyonkuru@1

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [#170947632](https://www.pivotaltracker.com/story/show/170947632)

#### Screenshots (if appropriate)

UI

Screen Shot 2020-05-26 at 09 04 34

ERROR VALIDATION

Screen Shot 2020-05-26 at 09 07 50

Successful Accommodation creation

Screen Shot 2020-05-26 at 09 09 25

#### Questions:

- N/A

### CODE REVIEW

**Review by Niyonsengaeric — COMMENTED** [review]
@Kagorora adds the travel admin credentials.

- it is not working
Screen Shot 2020-05-26 at 14 12 21

- consider also some code climate issues!!

**Review by Niyonsengaeric — APPROVED** [review]

**Review by T2Wil — APPROVED** [review]
Nice work

File: src/components/Nav.js [hunk 1]
```diff
@@ -1,28 +1,53 @@
...
+import jwt from 'jsonwebtoken';
 import logo from '../assets/images/logo@2x.png';
 import Languages from './Languages';
+import notificationIcon from '../assets/images/notification.svg';
+import AddProfile from './AddProfile';
+
+const profileImage = 'https://via.placeholder.com/150';
 
 const Nav = () => {
+  const tokenForTesting = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NiwiZW1haWwiOiJ0cmF2ZWxhZG1pbkBnbWFpbC5jb20iLCJyb2xlIjoidHJhdmVsYWRtaW4iLCJmaXJzdE5hbWUiOiJ0cmF2ZWxlciIsImxhc3ROYW1lIjoiSGltYmFyYSIsImlhdCI6MTU5MDU3NTM0N30.ktaH67UBQyE1V3g8LlJATchsA2jrWF8DyxK5cLarP4s';
```
Comment (kagabof) [inline src/components/Nav.js]: Why are we having this token?

File: src/components/NavSupplier.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import logo from '../assets/images/logo@2x.png';
+import Languages from './Languages';
+
+
+const profileImage = 'https://via.placeholder.com/150';
+dotenv.config();
+
+const NavAuth = () => {
```
[the remark this answers is not in the document]
Comment (kagabof) [reply src/components/NavSupplier.js:14]: You should fix this.

**Review by kagabof — CHANGES_REQUESTED** [review]

**Comment by kagabof** [comment]
Please check your test logs.

**Review by kagabof — APPROVED** [review]

**Pull request merged**